src/components/pages/user/components/completed-courses.tsx

```diff
@@ -33,7 +33,7 @@ const CompletedCourses: React.FC<{
         </div>
       ) : completedCourseStatus === 'error' ? (
         <span>There was an error fetching stats</span>
-      ) : !null || completeCourseData?.length === 0 ? (
```

Comment:
I think this condition is not correct - `!null` is also `true` so we will always get "You haven't finished any courses yet". Also I think we don't need optional chaining here because there is default `[]` value for `completeCourseData ` set up above.